### PR TITLE
Adding [Docs] Meshery Cloud Team Management and Notifications

### LINKED ISSUE #8056: [Docs] Meshery Cloud Team Management and Notifications
#### Current State
A section documenting the team management of Meshery cloud needs to be added  under Meshery-Extensions.
- Document the process of adding team members in Meshery cloud.
- Explain how team members are notified about team updates.
- Provide instructions for managing team settings and notifications.

#### Desired State

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by yash37158**
**Notes for Reviewers**
This is a draft pull request and needs a review and suggestions with the content @leecalcote @vishalvivekm @Chadha93 
This PR fixes #8056 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 33816b440a479c34494605382f24ccdc8949cce8 at: https://64b702ae27224512157f5c66--meshery-docs-preview.netlify.app

File: docs/pages/extensions/team-management/index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+* After creating the team, you become the admin or owner of the newly created team.
+
+* As the team admin or owner, you can now invite other team members by following the steps outlined earlier in the conversation.
+
+By following these steps, you can successfully join an existing team or create a new team within Meshery Cloud, allowing you to collaborate and contribute to the team's service mesh management activities.
+
+## Notifying Team Members about Updates
+
+In Meshery Cloud, team members are notified about team updates through various communication channels to ensure they stay informed and engaged. Notifications are used to disseminate important information and keep team members up to date with the latest developments.
```
Comment (leecalcote) [inline docs/pages/extensions/team-management/index.md:42]: Looking good, @yash37158. It's ideal to note Meshery Cloud's ability to integrate with third-party OAuth providers, too: GitHub and Google.

**Comment by vishalvivekm** [comment]
@yash37158 Let's discuss this on Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Comment by yash37158** [comment]
Sure @vishalvivekm

File: docs/pages/extensions/team-management/index.md (hunk 1 above)
Comment (yash37158) [reply docs/pages/extensions/team-management/index.md:42]: sure @leecalcote, I'll take that into consideration. also, I was looking forward to improving the notifications section, as I couldn't find resources on that topic. can you please help me with that too?

**Comment by l5io** [comment]
🚀 Preview for commit 7d86fd4bcb0d87c95d038902f14901efd4d75838 at: https://64b83e0d27e7681c8bbb4090--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 7d86fd4bcb0d87c95d038902f14901efd4d75838 at: https://64b83e0f1522e21bea23a2ff--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit c129bc059ca7471e0169ee4da228912abd5f0c08 at: https://64b83ecf08cd9e1e29b9500e--meshery-docs-preview.netlify.app

**Comment by yash37158** [comment]
^^ @leecalcote @Chadha93 @vishalvivekm

**Comment by l5io** [comment]
🚀 Preview for commit 1b87a3e71c1fb943ad714e2848b77671f34b30ad at: https://64b994e786e4fc38ea178f3d--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 1b87a3e71c1fb943ad714e2848b77671f34b30ad at: https://64b994f2c8e008385dd07456--meshery-docs-preview.netlify.app

File: docs/pages/extensions/team-management/index.md (hunk 1 above)
Comment (Ghat0tkach) [reply docs/pages/extensions/team-management/index.md:43]: @yash37158   , the team invites generally goes through the email.

**Comment by yash37158** [comment]
Hey @leecalcote @vishalvivekm are we good to merge this ?

**Comment by l5io** [comment]
🚀 Preview for commit fa7b0ae45f75880852252ddde2dd7348e65f1d41 at: https://64bec91e8af19f040c8b1479--meshery-docs-preview.netlify.app

**Review by leecalcote — APPROVED** [review]
Thank you, @yash37158 👍

**Pull request merged**